### PR TITLE
policies.md

### PR DESCRIPTION
**Pull request by kamleshjoshi8102**
/kind documentation  Some Typos Fixed in policies.md

### CODE REVIEW

File: docs/policies.md [hunk 1]
```diff
@@ -130,20 +130,20 @@ Read more about hostPath in the [official Kubernetes documentation](https://kube
...
 
-NodePorts are used as a way to expose a Service external to a cluster. Since NodePorts are a finite resource, this policy aims to control NodePort usage via an allow list model using an annotation on a Namespace to designate the allowable NodePort values/range.
+NodePorts are used as a way to expose a Service external to a cluster. Since NodePorts are a finite resource, this policy aims to control NodePort usage via an allow list model using annotation on a Namespace to designate the allowable NodePort values/range.
 
 The configuration should follow these recommended rules:
 
 The nodePort range annotation can be a single number, numbers separated by commas, a range split with a hyphen, or a combination of the three
 The nodePort annotation on the namespace should be "k8s.t-mobile.com/nodeportRange"
-Set the annotation to "na" if no nodePort range will to be set, that is seen as an exception value
+Set the annotation to "na" if no nodePort range will to be set, that is seen as an exceptional value
```
Comment (phenixblue) [inline docs/policies.md:147]: issue(blocking): `exception` in the original text is accurate within this context, but there are a couple of other items that could be addressed here:

```suggestion
Set the annotation to "na" if no nodePort range will not be set, that is seen as an exception value
```

Comment (phenixblue) [reply docs/policies.md:147]: chore (non-blocking): While it wasn't specifically called out in the original issue, there's a formatting issue on this section. 

The lines following the `The configuration should follow these recommended rules:` line should either have a newline in between each of them, or moved to an unordered list. Without this, the markdown is rendered as a continuous sentence with no punctuation.

**Review by phenixblue — CHANGES_REQUESTED** [review]
Thanks @kamleshjoshi8102 for contributing. I've left some inline comments and suggestions. please let mw know if you have nay questions.

**Comment by phenixblue** [comment]
@kamleshjoshi8102 Can you please rebase and address the statements in the PR review? If we don't see activity soon we will close this request out due to inactivity.

File: docs/policies.md (hunk 1 above)
Comment (kamleshjoshi8102) [reply docs/policies.md]: I can't understand what I have to change here. Can you please elaborate?

Comment (phenixblue) [reply docs/policies.md]: The highlighted changes in the suggestion above at a minimum. 

It's non-blocking, but would be appreciated to add a newline between each line following the `The configuration should follow these recommended rules:` line.

Right now it's like this:

```
The nodePort range annotation can be a single number, numbers separated by commas, a range split with a hyphen, or a combination of the three
The nodePort annotation on the namespace should be "k8s.t-mobile.com/nodeportRange"
Set the annotation to "na" if no nodePort range will to be set, that is seen as an exception value
Set the annotation to "na" if no nodePort range will not be set, that is seen as an exception value
```

And should be:

```
The nodePort range annotation can be a single number, numbers separated by commas, a range split with a hyphen, or a combination of the three

The nodePort annotation on the namespace should be "k8s.t-mobile.com/nodeportRange"

Set the annotation to "na" if no nodePort range will to be set, that is seen as an exception value

Set the annotation to "na" if no nodePort range will not be set, that is seen as an exception value
```

**Review by phenixblue — APPROVED** [review]
/lgtm

**Comment by phenixblue** [comment]
Thank you for your contributions!

**Comment by kamleshjoshi8102** [comment]
My Pleasure.!

**Pull request merged**